integration-tests/README.md

```diff
@@ -0,0 +1,107 @@
+  ```
+    # Build the Weblogic Kubernetes Operator binary 
+    mvn clean install 
+    # Run tests in sequential
+    mvn -pl -integration-tests -P integration-tests clean verify
+    # Run tests in parallel
+    mvn -DPARALLEL=true -DNUMBER_OF_THREADS=2 -pl -integration-tests -P integration-tests verify
+   ```
+## How to run individual test class(es)
+mvn -Dit.test=ItMiiDomain,ItMiiUpdateDomainConfig -pl integration-tests -P integration-tests verify
+
+## How to run a single test method in a test class 
+mvn -Dit.test="ItMiiUpdateDomainConfig#testMiiDeleteSystemResources" -pl integration-tests -P integration-tests verify
+
+## How to exclude test class(es) 
+mvn -Dit.test="!ItCrossDomainTransaction,!ItMiiUpdateDomainConfig" -pl integration-tests -P integration-tests verify
+
+## Environment variables to manage test execution 
+| Variable | Description | Default Value
+| --- | --- | --- |
+| SKIP_CLEANUP  | Test infrastructure removes all kubernetes objects created during test execution. To retain all such objects for triaging test failures this environment variable should be set to true. User need to run weblogic-kubernetes-operator/src/integration-tests/bash/cleanup.sh to clean up kubernetes objects after traiging.    | false
+| COLLECT_LOGS_ON_SUCCESS  | Test infra does not keep the diagnostic log for successful tests. To archive the diagnostic log for successful tests, this environment variable should be set to true. | false
+| RESULT_ROOT  | Root directory for the integration test results and artifacts generated during test execution. | /tmp/it-testsresults
+| BASE_IMAGES_REPO  | The repository URL to download WebLogic and FMW image. Make sure you have access to this repository. Other supported repository is phx.ocir.io (OCIR)  | container-registry.oracle.com(OCR)
+| WEBLOGIC_IMAGE_NAME  | Name of the WebLogic image in the chosen repository.| middleware/weblogic (OCR) 
+| WEBLOGIC_IMAGE_TAG  | The tag for WebLogic base image. Generally, it represents the WebLogic version with JDK and/or installation type. Possible values are 12.2.1.3, 12.2.1.3-dev, 12.2.1.4-slim, 14.1.1.0-11 or 14.1.1.0-8. Please check the repository for the availability of these images. | 12.2.1.4
+| FMWINFRA_IMAGE_TAG  | The tag for Fusion Middleware Infrastructure base image. Generally, it represents the FMW version. Possible values are 12.2.1.3, 12.2.1.4. Please check the repository for the availability of these images. | 12.2.1.4
+| FMWINFRA_IMAGE_NAME  | Name of the Fusion Middleware Infrastructure image in the chosen repository.| middleware/fmw-infrastructure (OCR) 
+
+## Logging/Archiving
+
+On completion of integration test execution, the results are archived in a directory based on environment variable  LOG_DIR  (the default value is /tmp/it-testsresults/it-diagnostic) and runtime artifacts are available in a directory based on environment variable RESULTS_ROOT (the default value is /tmp/it-testresults) . 
```

Comment:
On completion of integration test execution, -> After the completion of running the integration tests,
directory based -> directory-based
LOG_DIR -> `LOG_DIR` (code font)
/tmp/it-testsresults/it-diagnostic -> `/tmp/it-testsresults/it-diagnostic` (code font)
RESULTS_ROOT -> `RESULTS_ROOT` (code font)
/tmp/it-testresults -> `/tmp/it-testresults` (code font)

Reply:
done

Reply:
Slight confusion here. Choices are: a directory-based environment variable OR a directory based on the environment variable (I think the latter is what you want)

Reply:
[modified]  After the completion of running the integration tests, the results are archived in a directory-based environment variable

Reply:
How about this one: and runtime artifacts are available in a directory based on environment variable
Should it be: and runtime artifacts are available in a directory-based environment variable

Reply:
[ Modified ] After the completion of running the integration tests, the results are archived in a directory-based environment variable `LOG_DIR` (the default value is `/tmp/it-testsresults/diagnostic`) and runtime artifacts are available in a directory-based environment variable `RESULTS_ROOT` (the default value is `/tmp/it-testresults`).